pkg/modules/application_mobility.go

```diff
@@ -92,10 +92,10 @@ const (
 	InitContainerImage = "<INIT_CONTAINER_IMAGE>"
 	//AccessContents - contents of the object store secret
 	AccessContents = "<CRED_CONTENTS>"
-	//AccessKeyID - contains the aws access key id
-	AccessKeyID = "<KEY_ID>"
-	//AccessKey - contains the aws access key
-	AccessKey = "<KEY>"
+	//AKeyID - contains the aws access key id
```

Comment:
This was already changed by @jooseppi-luna 
Please confirm with him and also there were more occurrences of this variable
Verify it

Reply:
I would just grep and make sure you get everything. Also should run all unit tests and build an image and run e2e tests after any changes

Reply:
done